example/material.json

```diff
@@ -64,6 +64,14 @@
             "element": "Si",
             "name": "elemental_ratio",
             "value": 0.71
+        },
+        {
+            "name": "inchi",
+            "inchi": "1S/Si3/c1-3-2"
+        },
+        {
+            "name": "inchi_key",
+            "inchi_key": "YBILFZWSTSXODU-UHFFFAOYSA-N"
```

Comment:
Does it make more sense to have these two parameters together instead of as two separate parameters. For example as
```{                                                                                         ```
```     "name": "International Chemical Identifier",             ```
```     "inchi": "1S/Si3/c1-3-2",                                             ```
```    "inchi_key": "YBILFZWSTSXODU-UHFFFAOYSA-N"```
```}                                                                                         ```

Reply:
1. If all other elements of `derivedProperties` have the same schema with `name` and `value`, shouldn't we just use that??
2. This material example should not have `inchi`

Reply:
1. That's true. I switched the dict key from `inchi`/`inchi_key` to `value`.
2. Would it make sense to still have an `inchi` placeholder in the schema, to follow the `derivedProperties` format. But have it labeled as `Not Available` or `Not Applicable` instead? (see current PR example). 
3. Does it make sense to have a `molecule.js` file for non-periodic systems instead? The `material.json` file does have a line for `isNonPeriodic` to be defined, so that could be used to distinguish between a material/molecule rather than two separate schema.